### PR TITLE
Remaining lock transition sounds.

### PR DESCRIPTION
**Pull request by adays1234**
Monophonic bgnlock untested.

Polyphonic bgnlock and endlock tested by copying polyphonic clsh sounds.

### CODE REVIEW

File: sound/hybrid_font.h [hunk 1]
```diff
@@ -173,7 +173,11 @@ class HybridFont : public SaberBase {
 	  drag.files_found()) {
 	PlayMonophonic(&drag, &drag);
       } else if (lockup.files_found()) {
-	PlayMonophonic(&lockup, &lockup);
+        if (bgnlock.files_found()) {
+          PlayMonophonic(&bgnlock, &lockup);
+        } else {
+	  PlayMonophonic(&lockup, &lockup);
```
Comment (profezzorn) [inline sound/hybrid_font.h]: indentation is off

File: sound/hybrid_font.h [hunk 2]
```diff
@@ -182,7 +186,12 @@ class HybridFont : public SaberBase {
 	e = &drag;
       }
       if (!lock_player_) {
-	lock_player_ = PlayPolyphonic(e);
+        if (bgnlock.files_found()) {
+          lock_player_ = PlayPolyphonic(&bgnlock);
+        } else {
+	  lock_player_ = PlayPolyphonic(e);
```
Comment (profezzorn) [inline sound/hybrid_font.h]: indentation is off

Comment (adays1234) [reply sound/hybrid_font.h]: Not sure how that happened but oops! Lemme fix.

File: sound/hybrid_font.h (hunk 1 above)
Comment (adays1234) [reply sound/hybrid_font.h]: Not sure how that happened but oops! Lemme fix.

**Comment by adays1234** [comment]
Requested white space fixes made

**Pull request merged**